packages/cf-component-tabs/src/Tabs.js

```diff
@@ -64,7 +64,9 @@ class Tabs extends React.Component {
             })}
           </ul>
         </Viewport>
-        {this.props.children}
+        {this.props.children.find(child => {
```

Comment:
1. Could I just change the proptype to be a array of elements?
2. Not sure. I'll check.